### ISSUE TITLE
doesn't support Night Mode

### ISSUE THREAD

**Issue by dmalan** [cs50/harvard.cs50.droplet#19]
We need to use Night Mode in Sanders for visibility's sake on stage.

![screen shot 2016-08-25 at 10 04 41 pm](https://cloud.githubusercontent.com/assets/788678/17991831/41094efe-6b10-11e6-9aa4-86f76e225130.png)

**Comment by dabbler0** [comment]
I am pushing a commit that fixes the Droplet animation so that it animates smoothly between light and dark text. This should allow presentations that use night mode for text code.

However, I can't make Droplet work properly with a white-on-dark blocks theme without rethinking all of the Droplet colors. Luke and I picked these background colors, cursor, and highlights specifically to be readable with black text, and white text is really illegible.

Night mode for blocks mode is something I could conceivably implement if we looped in Luke and I had a little bit more time.
